### PR TITLE
community/logcheck: add busybox compatibility / move to community

### PR DESCRIPTION
**Pull request by itoffshore**
I've been running `logcheck` for 6 weeks with `syslog-summary` (https://github.com/alpinelinux/aports/pull/269) without any problems. Tested with both busybox `syslog` & `socklog`.

### CODE REVIEW

File: community/logcheck/logcheck.cron.sample.d [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ 
+@reboot         logcheck    if [ -x /usr/sbin/logcheck ]; then nice -n10 /usr/sbin/logcheck -R; fi
+2 * * * *       logcheck    if [ -x /usr/sbin/logcheck ]; then nice -n10 /usr/sbin/logcheck; fi
+
+# EOF
+
+
+## busybox crond reboot script [ /etc/periodic/reboot/logcheck ]
+
+#!/bin/sh
```
Comment (jirutka) [inline community/logcheck/logcheck.cron.sample.d]: How do these script snippets inside cron file work? I’ve never seen it.

File: community/perl-ipc-signal/APKBUILD [hunk 2]
```diff
@@ -1,37 +1,33 @@
...
-makedepends="perl-dev $cpanmakedepends"
+depends=""
+makedepends="perl-dev"
 subpackages="$pkgname-doc"
 source="http://search.cpan.org/CPAN/authors/id/R/RO/ROSCH/$_pkgreal-$pkgver.tar.gz"
-
-_builddir="$srcdir/$_pkgreal-$pkgver"
+builddir="$srcdir/$_pkgreal-$pkgver"
 
 prepare() {
```
Comment (jirutka) [inline community/perl-ipc-signal/APKBUILD]: Please add `default_prepare || return 1` at top of custom `prepare` function.

File: community/perl-mime-construct/APKBUILD [hunk 3]
```diff
@@ -1,36 +1,34 @@
...
 depends="perl $cpandepends"
-makedepends="perl-dev $cpanmakedepends"
+makedepends="perl-dev"
 subpackages="$pkgname-doc"
 source="http://search.cpan.org/CPAN/authors/id/R/RO/ROSCH/$_pkgreal-$pkgver.tar.gz"
-
-_builddir="$srcdir/$_pkgreal-$pkgver"
+builddir="$srcdir/$_pkgreal-$pkgver"
 
 prepare() {
```
Comment (jirutka) [inline community/perl-mime-construct/APKBUILD]: Please add `default_prepare || return 1` at top of custom `prepare` function.

File: community/perl-proc-waitstat/APKBUILD [hunk 4]
```diff
@@ -1,37 +1,34 @@
...
 depends="$cpandepends"
-makedepends="perl-dev $cpanmakedepends"
+makedepends="perl-dev"
 subpackages="$pkgname-doc"
 source="http://search.cpan.org/CPAN/authors/id/R/RO/ROSCH/$_pkgreal-$pkgver.tar.gz"
-
-_builddir="$srcdir/$_pkgreal-$pkgver"
+builddir="$srcdir/$_pkgreal-$pkgver"
 
 prepare() {
```
Comment (jirutka) [inline community/perl-proc-waitstat/APKBUILD]: Please add `default_prepare || return 1` at top of custom `prepare` function.

File: community/logcheck/logcheck.cron.sample.d (hunk 1 above)
Comment (itoffshore) [inline community/logcheck/logcheck.cron.sample.d]: Those lines are from the original sample by @ncopa - I think they may work with another cron (not busybox `crond`).

Let me know if you want if removed - my busybox version has been running ok for weeks now

Comment (jirutka) [inline community/logcheck/logcheck.cron.sample.d]: I’d just like to know how it works, when it is executed; I’m curious and can’t figure out right keywords to google it. :)

If it doesn’t work on Busybox crond, then it should be mentioned in the comment.

**Comment by itoffshore** [comment]
`default_prepare` added to the 3 `perl` pkgs. 

I also changed the `cron` script / example a little so they work with `linux-pam`.

**Comment by itoffshore** [comment]
this no longer uses the busybox initscript change that was not accepted. 

the sample file shows an `/etc/local.d` script

**Pull request closed without merging**